web/src/pages/Settings/Groups/Groups.jsx

```diff
@@ -31,27 +32,31 @@ const GroupListItem = ({ group, zones }) => {
     };
 
     return (
-        <ListItem
-            name={group.name}
-            onClick={() => {
-                setModalOpen(true);
-            }}
-        >
-            <div className="groups-group-icon">
-                <SpeakerGroup fontSize="inherit" />
-            </div>
-            {modalOpen && (
-                <GroupModal
-                    group={group}
-                    zones={zones}
-                    onClose={() => {
-                        setModalOpen(false);
-                    }}
-                    del={deleteGroup}
-                    apply={editGroup}
-                />
-            )}
-        </ListItem>
+        <>
+            <ListItem
+            style={{fontSize: "2rem"}}
+                onClick={() => {
+                    setModalOpen(true);
+                }}
+            >
+                <div className="groups-group-icon">
+                    <SpeakerGroup fontSize="inherit" />
+                </div>
+                {group.name}
+                {modalOpen && (
+                    <GroupModal
+                        group={group}
+                        zones={zones}
+                        onClose={() => {
+                            setModalOpen(false);
+                        }}
+                        del={deleteGroup}
+                        apply={editGroup}
+                    />
+                )}
+            </ListItem>
+            <Divider component="li" />
```

Comment:
honest question; does it look better without the divider and just bigger margins? that's what seemed to work well on the stream page, but maybe these cases are different?

Reply:
> that's what seemed to work well on the stream page

The stream page also has these dividers, they're added here due to the issue being "Make all the errant stylings in the settings tabs fall in line"
I think you'll have to clarify somewhat when you're back to show me what you mean

Reply:
personally I think dividers should separate sections, not individual items in a list. imo visually it reads more like a list if there isn't separation. for example, [material design doesn't use a divider but only whitespace between list items](https://m3.material.io/components/lists/overview).

Reply:
That page has 1 instance of being whitespace separated:
![image](https://github.com/user-attachments/assets/7532b5bb-20db-4837-9476-316623ddf792)

And two instances of divider-separation:
![image](https://github.com/user-attachments/assets/b5ed44a2-633c-4446-bc93-6939c272999d)
![image](https://github.com/user-attachments/assets/cef14eca-36e4-48a6-97d0-fedb4e01f36c)


So it seems to be a matter of personal preference, I may say we bring this up in todays meeting to poll for group preference

Reply:
i think i'd prefer to see a before and after of AmpliPi with & without dividers, if we're gonna compare anything. it looked better in the one case where I removed them; not convinced that's true here too, which is why i asked instead of suggested.

Reply:
Before:
![image](https://github.com/user-attachments/assets/6cd62ccf-ca3c-458c-8a03-b5e738234c51)

After:
![Screenshot 2024-07-22 130914](https://github.com/user-attachments/assets/6eac0017-fd0e-448c-a76f-08e75e803a38)

Reply:
I like the lines!

Reply:
lines it is then :sunglasses: